data/hire.json

```diff
@@ -0,0 +1,12 @@
+// Add yourself to this via a pull-request to the source repo at
+// https://github.com/lrug/lrug.org/pulls
+// Do keep the list sorted by last name when you do so.
+// We're using microformats as the keys here, although we currently only
+// render out fn and url. Feel free to add more keys and we'll update the
+// renderer.
+[
+  {
+    "fn":"James R. McCarthy, LetyCo",
+    "url":"http://lety.co/"
+  }
```

Comment:
Surely if the user feels strongly about this they can indicate so on the site they link to. I fail to see a good reason to include more than a link to a site of your choice.


At first I thought that indicating whether you we're presently available was a good idea, but then realised that this info would go stale without constant updating on the users part, which would bring in the requirement for there to be an app or system allowing users to easily keep their status updated.

Given the small numbers involved in this it's not really worthwhile.